pom.xml

```diff
@@ -1,7 +1,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<groupId>org.openhab.jmdns</groupId>
```

Comment:
> So in essence, does that mean that the original javax.jmdns project is deprecated?

Yes, there won't be any further releases done from Sourceforge (and thus the javax.jmdns Maven GroupId) anymore. All further evolution is done right here at Github.